pages/1.12/cli/plugins/index.md

```diff
@@ -12,14 +12,14 @@ enterprise: false
 
 Plugins extend the functionality of the CLI for cluster specific actions.
 
-When setting up a cluster, the CLI will install the core plugin by default and, if applicable, the enterprise plugin.
-
 Plugins are cluster-specific, when switching between different clusters the CLI will automatically use the proper set of plugins installed for that cluster.
 
 # Core CLI Plugin
 
 Commands that are specific to a DC/OS version are packaged into a plugin called `dcos-core-cli`. It gets automatically installed when running `dcos cluster setup` unless the `--no-plugin` flag is used. The subcommands included in this plugin are:
 
+In version 0.7, the CLI has the 1.12 Core plugin included with it which it will automatically install for any cluster that it's attached to.
```

Comment:
I don't understand this sentence.

Reply:
This is talking about the 1.12 Core bundling that we're doing.

Reply:
Shouldn't `the 1.12 Core plugin included with it which it will automatically install for any cluster that it's attached to.` be `the 1.12 Core plugin included which will automatically be installed for any cluster it's attached to.`?